src/clients/c++/image_client.cc

```diff
@@ -388,6 +445,7 @@ ParseModel(
 
     if (dim > 1) {
       non_one_cnt++;
+      *op_size = dim;
```

Comment:
Not sure what op_size is. Looks like the size of the output tensor. If so needs a better name, like output_size